src/registrar/admin.py

```diff
@@ -2864,23 +2864,196 @@ def save_model(self, request, obj, form, change):
+        ("Type of organization", {"fields": ["organization_type", "federal_type"]}),
+        (
+            "Organization name and mailing address",
+            {
+                "fields": [
+                    "federal_agency",
+                    "state_territory",
+                    "address_line1",
+                    "address_line2",
+                    "city",
+                    "zipcode",
+                    "urbanization",
+                ]
+            },
+        ),
+        ("Senior official", {"fields": ["senior_official"]}),
+    ]
+
+    # NOT all fields are readonly for admin, otherwise we would have
+    # set this at the permissions level. The exception is 'status'
+    analyst_readonly_fields = [
+        "federal_type",
+    ]
 
     list_display = ("organization_name", "federal_agency", "creator")
     search_fields = ["organization_name"]
     search_help_text = "Search by organization name."
     readonly_fields = [
-        "creator",
+        # This is the created_at field
+        "created_on",
+        # Custom fields such as these must be defined as readonly.
+        "domains",
+        "domain_requests",
+        "suborganizations",
+        "portfolio_type",
     ]
 
+    def created_on(self, obj: models.Portfolio):
+        """Returns the created_at field, with a different short description"""
+        # Format: Dec 12, 2024
+        return obj.created_at.strftime("%b %d, %Y") if obj.created_at else "-"
+
+    created_on.short_description = "Created on"  # type: ignore
+
+    def portfolio_type(self, obj: models.Portfolio):
+        """Returns the portfolio type, or "-" if the result is empty"""
+        return obj.portfolio_type if obj.portfolio_type else "-"
+
+    portfolio_type.short_description = "Portfolio type"  # type: ignore
+
+    def suborganizations(self, obj: models.Portfolio):
+        """Returns a comma seperated list of links for each related suborg"""
+        queryset = obj.get_suborganizations()
+        sep = '<div class="display-block margin-top-1"></div>'
+        return self.get_field_links_as_csv(queryset, "suborganization", seperator=sep)
+
+    suborganizations.short_description = "Suborganizations"  # type: ignore
+
+    def domains(self, obj: models.Portfolio):
+        """Returns a comma seperated list of links for each related domain"""
+        queryset = obj.get_domains()
+        sep = '<div class="display-block margin-top-1"></div>'
+        return self.get_field_links_as_csv(
+            queryset, "domaininformation", link_info_attribute="get_state_display_of_domain", seperator=sep
+        )
+
+    domains.short_description = "Domains"  # type: ignore
+
+    def domain_requests(self, obj: models.Portfolio):
+        """Returns a comma seperated list of links for each related domain request"""
+        queryset = obj.get_domain_requests()
+        sep = '<div class="display-block margin-top-1"></div>'
```

Comment:
Blocking: This pattern is non-semantic and is probably accruing tech debt until design does an accessibility audit here. These should be <li> inside of an unstyled <ul>